docs/api/commands/writefile.mdx

```diff
@@ -72,6 +72,21 @@ parameter and include encoding there. This is the same behavior as
 
 :::
 
+### Requirements [<Icon name="question-circle"/>](/guides/core-concepts/introduction-to-cypress#Chains-of-Commands)
+
+- `cy.writeFile()` ignores the subject it is given.
+- `cy.writeFile()` requires the file be successfully written to disk. Anything
+  preventing this such as OS permission issues will cause it to fail.
+
+### Assertions [<Icon name="question-circle"/>](/guides/core-concepts/introduction-to-cypress#Assertions)
+
+- `cy.writeFile()` has no implicit assertions.
```

Comment:
lets link to the implicit assertion documentation for users unware of what this means

Reply:
Added links to each command page.

I added the link to `/guides/core-concepts/introduction-to-cypress#Implicit-Assertions` - in this branch the URL is `/guides/core-concepts/introduction-to-cypress#Default-Assertions`, but I'm renaming the section `Implicit Assertions` in https://github.com/cypress-io/cypress-documentation/pull/5081/files, so using the new name here to avoid extra work later.